manifests/05_operator_clusterrole.yaml

```diff
@@ -82,24 +82,4 @@ rules:
   - clusterroles
   - clusterrolebindings
   verbs:
-  - watch
-  - list
-  - get
-  - create
-  - delete
-  - patch
-  - update
-- apiGroups:
-  - groupsnapshot.storage.k8s.io
-  resources:
-  - volumegroupsnapshots
-  - volumegroupsnapshotclasses
-  - volumegroupsnapshotcontents
-  verbs:
-  - watch
-  - list
-  - get
-  - create
-  - delete
-  - patch
-  - update
+  - "*"
```

Comment:
We used to use wildcards in our roles, but were told to remove them and list the permissions explicitly